backend/src/main/java/fk/prof/backend/http/handler/RecordedProfileRequestHandler.java

```diff
@@ -27,7 +39,14 @@ public RecordedProfileRequestHandler(RoutingContext context, CompositeByteBufInp
 
   @Override
   public void handle(Buffer requestBuffer) {
-//    System.err.println(String.format("buffer=%d, chunk=%d", runningBuffer.length(), requestBuffer.length()));
+    //    try { logger.debug(String.format("buffer=%d, chunk=%d", inputStream.available(), requestBuffer.length())); } catch (Exception ex) {}
+    histChunkSize.update(requestBuffer.length());
+    long currentTime = System.nanoTime();
+    if(chunkReceivedTime != null) {
+      tmrChunkIdle.update(currentTime - chunkReceivedTime, TimeUnit.NANOSECONDS);
+    }
+    chunkReceivedTime = currentTime;
+
     if (!context.response().ended()) {
       inputStream.accept(requestBuffer.getByteBuf());
       try {
```

Comment:
All API related metrics are covered by vertx as default. There is a catch. vertx gives throughput timer for every API but statuscode metrics are across all APIs. we might choose to use cosmos-tail plugin over our access.logs to generate whatever remaining metrics that are missing at API level